_config.yml

```diff
@@ -8,6 +8,9 @@
 # jekyll-redirect-from, and we don't need those to be fully qualified.
 pulumi_url: https://pulumi.io
 
+# Needed for jekyll-sitemap
+url: "https://pulumi.io"
```

Comment:
Now that we need this for the sitemap, would you mind adding a commit that removes the `pulumi_url` variable above along with the comment, and replace it's use in `head.html` (e.g. `site.pulumi_url` => `site.url`)?

Reply:
Made this change